hardware/Makefile

```diff
@@ -234,12 +234,12 @@ patch_NVMe:
 	fi
 
 action_hw: prepare_logs
-	@echo -e "[CONFIG ACTION HW....] start `date +"%T %a %b %d %Y"`";
+	@echo -e "[COMPILE ACTION HW...] start `date +"%T %a %b %d %Y"`";
 	@if [ ! -d "$(ACTION_ROOT)" ]; then \
 		echo "                        Error: Path to ACTION_ROOT is not defined properly"; exit -1; \
 	fi
 	@echo "Calling make -C $(ACTION_ROOT) hw" > $(LOGS_DIR)/action_make.log
-	@$(MAKE) -C $(ACTION_ROOT) hw
+	@$(MAKE) -C $(ACTION_ROOT) hw >> $(LOGS_DIR)/action_make.log;
```

Comment:
The make process should be able to produce console messages. We need to find another way to suppress unwanted messages produced by HLS compile. (Again, this is not only a target for HLS actions)

Reply:
ok modified so that only hls is impacted